lib/openstudio-standards/standards/necb/ECMS/nv.rb

```diff
@@ -0,0 +1,187 @@
+class ECMS
+
+  def apply_nv(model:, nv_type:, nv_comfort_model:, nv_opening_fraction:, nv_Tout_min:, nv_Delta_Tin_Tout:)
+
+    ##### If any of users' inputs are nil/false, do nothing.
```

Comment:
Do you have an option for 'NECB_Default'

Reply:
Added.